platformcomponents/desktop/content-list-item.json

```diff
@@ -0,0 +1,25 @@
+{
+  "content-list-item": {
```

Comment:
content-file-list-item

Reply:
Pushed commit
`Renamed content-list-item to content-file-list-item`
Changing this conversation to Resolved.